wp-content/themes/pub/wporg-learn-2020/front-page.php

```diff
@@ -32,15 +32,30 @@
 			</div>
 		</section>
 
-		<section class="about-training">
-			<div class="getin">
-				<div class="graphic"><span class="dashicons dashicons-welcome-learn-more"></span></div>
-				<div>
-					<h2 class="h3"><?php esc_html_e( 'Get Involved', 'wporg-learn' ); ?></h2>
-					<p><?php esc_html_e( 'Want to get involved in creating the content for Learn WordPress?', 'wporg-learn' ); ?></p>
-					<a href="//learn.wordpress.org/contribute"><?php esc_html_e( 'Learn how to contribute', 'wporg-learn' ); ?></a>
-				</div>
+		<section>
+			<div class="row align-middle between section-heading">
+				<h2 class="h4 section-heading_title"><?php esc_html_e( 'Recent Courses', 'wporg-learn' ); ?></h2>
+				<a class="section-heading_link" href="/courses"><span aria-hidden="true"><?php esc_html_e( 'View All »', 'wporg-learn' ); ?></span><span class="screen-reader-text"><?php esc_html_e( 'View All Courses', 'wporg-learn' ); ?></span></a>
```

Comment:
Re the double arrow: I think translators prefer that you either use an entity like `&raquo;`, or leave it out entirely. Could this be simplified by just using `View All Courses` for visible and screen reader alike?

Reply:
Yeah, I'll make that change. I had copied this from the workshops block below this, so I'll update that one to also remove the double arrow.